### PR TITLE
Separate embedded resources from source code

### PR DESCRIPTION
**Pull request by mr-martian**
This PR renames `apertium-init.py` to `main.py` and has `make` generate `apertium-init.py` from `main.py` and the various `updateBootstraper.py` scripts.

My reason for proposing this is that having the compressed file data in the source code is philosophically ugly and requires every open PR to be redone every time any other change is pushed, even if it's unrelated.

According to @TinoDidriksen, it should be straightforward to replace the download link in the readme with one that points to a copy of the script in nightly builds so that nothing needs to change externally.

### CODE REVIEW

**Comment by mr-martian** [comment]
I had forgotten `.gitignore`.

I was planning to also incorporate updating the download instructions both here and on the wiki, I just didn't know what the new download link would be yet.

**Comment by mr-martian** [comment]
If we go ahead with this, then when I know what the new link would be, my plan is to combine all changes into the first commit, except for deleting the data lines from the source file (so that git can keep track of the renaming).

Also, so that they don't get lost, here are all the wiki pages that would need download links changed:
- https://wiki.apertium.org/wiki/Apertium-init
- https://wiki.apertium.org/wiki/How_to_bootstrap_a_new_pair
- https://wiki.apertium.org/wiki/The_quick_and_dirty_guide_to_making_a_new_language_pair
- https://wiki.apertium.org/wiki/Як_створити_нову_мовну_пару

**Comment by TinoDidriksen** [comment]
[apertium-init is now packaged](https://github.com/apertium/apertium-packaging/commit/390a76451e644c4b23df807b6279c27554276d7a) and will appear on https://apertium.projectjj.com/apt/nightly/ and https://apertium.projectjj.com/apt/release/ after successful builds. Right now there's a nightly build.

This also means people can get apertium-init via nightly repo and apt-get.

**Comment by TinoDidriksen** [comment]
Prettier URL: https://apertium.org/apertium-init

**Comment by mr-martian** [comment]
I've got everything in 2 commits now and I also fixed a typo that caused `make test` to fail on newer versions of mypy, so this should be ready to merge if the concept is agreeable.

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Bootstrap Apertium language modules and pairs using `apertium-init.py`.
 
 ## Usage
 
-First, <a href="https://raw.githubusercontent.com/apertium/bootstrap/master/apertium-init.py" download>download the script</a>
+First, <a href="https://apertium.projectjj.com/apt/nightly/apertium-init" download>download the script</a>
```
Comment (sushain97) [inline README.md:12]: Can we use the apertium.org URL here?

**Review by sushain97 — COMMENTED** [review]
One nit.

**Comment by sushain97** [comment]
Reminder to self: Use rebase and merge for this PR, not squash and merge.

**Review by sushain97 — APPROVED** [review]

**Comment by mr-martian** [comment]
Thanks @sushain97!

**Pull request merged**